include/votca/csg/topology.h

```diff
@@ -46,7 +49,9 @@ namespace csg {
 class Interaction;
 
 using MoleculeContainer = std::vector<Molecule *>;
-using BeadContainer = std::vector<Bead *>;
+// Stable vectors are necessary to be sure that the Bead pointers that point
+// to addresses in the vector are not invalidated as the vector grows.
```

Comment:
I mean the other way to do it is just build the vector of beads first afterwards it is stable.